src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -42,6 +44,12 @@ public ITaskItem[] AssembliesToPublish
         {
             get { return _assembliesToPublish.ToArray(); }
         }
+        [Output]
+        public ITaskItem[] PackagesPublished
```

Comment:
This is kind of a bad name for the value that is being returned.  The name says "Packages", but we are returning "Assemblies".